runtime/src/main/java/com/xatkit/plugins/moodle/platform/utils/MessageObject.java

```diff
@@ -20,6 +21,11 @@
      */
     private String userId;
 
+    /**
+     * The session data of the user who sent the message.
+     */
+    private String currentCourseAccess;
```

Comment:
Wouldn't it be better if it was called `currentCourseId`? It would be clearer what data it actually stores